teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
+| Health Care VAOS - Appointment Type | 71 | Hit | Appointment type (either `confirmed` or `pending`) for a VAOS apointment |
+| Health Care VAOS - Type of Care | 72 | Hit | Answer to the question asking users to `choose the type of care you need`. Options include: `primary care`, `mental health`, `optometry`, `COVID-19 vaccine`, etc. Pages this CD is set: www.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment |
+| Health Care VAOS - Flow | 73 | Hit | Type of flow for a VAOS request. Options include: `va-request`, `direct`, and `cc-request` |
+| Education - 22-1995 - Exhausted All Benefits | 74 | Hit | **No data is being collected** |
+| Health Care VAOS - Facility Type | 75 | Hit | Facility type a user selects in the VAOS flow. Options incude: `va` or `cc` |
+| Health Care VAOS - Reason for Appointment | 76 | Hit | **No data is being collected**. Answer to the question asking users to `choose a reason for your appointment`. Options include: `routine or follow-up visit`, `I have a new medical issue`, `I have a concern or question about my medication`, or `My reason isn't listed here`. |
+| Frequently Asked Question | 77 | Hit | CD is set to the FAQ question that houses the hyperlink a user clicks on on this page: www.va.gov/coronavirus-veteran-frequently-asked-questions/ (for example: `Should I get a flu shot this year?`) |
+| FAQ Section | 78 | Hit | CD is set to the FAQ section that the user selected a specific question in. For example, this CD would be set to `Coronavirus-specific concerns` if a user were to select a hyperlink under the question, "Should I get a flu shot this year?". |
+| Education - YR - Filter - Contribution Amount | 79 | Hit | Contribution amount CD is set to either `(not set)` or `unlimited` |
+| Education - YR - Filter - Number Of Students | 80 | Hit | Contribution amount CD is set to either `(not set)` or `unlimited` |
+| Education - School Name | 81 | Hit | CD is set to the school name entered into the text box "Enter full school name" found on this page: https://www.va.gov/education/yellow-ribbon-participating-schools/. Many of the `Education - School Name` CD entries are set to `(not set)` |
+| Education - School State | 82 | Hit | CD is set to the state entered into the box "State or territory" found on this page: https://www.va.gov/education/yellow-ribbon-participating-schools/. Many of the `Education - School Name` CD entries are set to `(not set)` |
+| Education - School City | 83 | Hit | CD is set to the city entered into the box "City" found on this page: https://www.va.gov/education/yellow-ribbon-participating-schools/. Many of the `Education - School Name` CD entries are set to `(not set)` |
+| [old] Education - YR - # of Search Results | 84 | Hit | CD is set to an integer matching the number of search results of a search on this page: www.va.gov/education/yellow-ribbon-participating-schools/ |
```

Comment:
If this is not actually collecting anymore let's change to unused in GA and mark this here